pss/trojan/message.go

```diff
@@ -108,6 +111,31 @@ func (m *Message) Wrap(targets [][]byte) (chunk.Chunk, error) {
 	return chunk, nil
 }
 
+// Unwrap attemps to determine whether the given chunk is a trojan chunk
+// it will return the resulting message if the unwrapping is successful, and an error otherwise
+// TODO: move this to PSS API?
+func Unwrap(c chunk.Chunk) (*Message, error) {
```

Comment:
i would call this on a byte slice already, not  a chunk

Reply:
are you sure? i liked the symmetry of having `Wrap` return a `chunk.Chunk` and then `Unwrap` taking one as param